src/nyc_trees/nyc_trees/context_processors.py

```diff
@@ -3,11 +3,44 @@
 from __future__ import unicode_literals
 from __future__ import division
 
-from django.conf import settings
+from django.conf import settings as s
 
+from calendar import timegm
+from datetime import datetime
 
-def nyc_bounds(request):
-    return {'NYC_BOUNDS_XMIN': float(settings.NYC_BOUNDS[0]),
-            'NYC_BOUNDS_YMIN': float(settings.NYC_BOUNDS[1]),
-            'NYC_BOUNDS_XMAX': float(settings.NYC_BOUNDS[2]),
-            'NYC_BOUNDS_YMAX': float(settings.NYC_BOUNDS[3])}
+from django.utils.timezone import make_aware, utc
+
+from apps.survey.models import Blockface, Survey
+
+
+def settings(request):
+    return {
+        "tiler_url": s.TILER_URL,
+        "db_name": s.DATABASES['default']['NAME'],
+        'nyc_bounds': {
+            'xmin': float(s.NYC_BOUNDS[0]),
+            'ymin': float(s.NYC_BOUNDS[1]),
+            'xmax': float(s.NYC_BOUNDS[2]),
+            'ymax': float(s.NYC_BOUNDS[3])
+        }
+    }
+
+
+def tiler_cache_busters(request):
+    blockface_updated_at = _get_last_updated_datetime(Blockface)
```

Comment:
Consider checking if this is an AJAX request (https://docs.djangoproject.com/en/dev/ref/request-response/#django.http.HttpRequest.is_ajax) before making these DB calls.

We don't need the tile cache busters for HTML partials, and it's probably a safe bet that an AJAX request will mean we're returning an HTML partial, not a full page that extends `base.html`.

Something like this:
```python
def tiler_cache_busters(request):
    if request.is_ajax():
        return {}
    ...
```